bob/participant-info.json

```diff
@@ -43,29 +43,21 @@
       "title": "EMV Participant Information",
       "type": "object",
       "required": [
-        "iin"
+        "aid"
       ],
       "properties": {
-        "iin": {
-          "title": "Issuer Identification Numbers",
-          "type": "array",
-          "items": {
-            "type": "integer",
-            "example": 97523124
-          }
+        "aid": {
+          "title": "Application Identifier",
+          "type": "string",
+          "pattern": "^[0-9A-Fa-f]{10,32}$",
+          "example": "A000000838"
         },
         "keys": {
           "title": "CA Public Keys",
           "type": "array",
           "items": {
             "type": "object",
             "properties": {
-              "rid": {
```

Comment:
Keep RID in key-array to follow the way CA-keys are normally defined

Reply:
7b32ad69c3ffc3b1606942e1b3cbd00779f0d8e3